### ISSUE TITLE
Error when opening website > My Medical:  "'website' object has no attribute 'render'"

### ISSUE THREAD

**Issue by jcdrubay** [LasLabs/vertical-medical#77]
After installing almost every module.

```
Traceback (most recent call last):
  File "/opt/openerp/code/openerp/project/trobz-modules/odoo/addons/website/models/ir_http.py", line 264, in _handle_exception
    response = super(Http, cls)._handle_exception(exception)
  File "/opt/openerp/code/openerp/project/trobz-modules/odoo/odoo/addons/base/ir/ir_http.py", line 169, in _handle_exception
    return request._handle_exception(exception)
  File "/opt/openerp/code/openerp/project/trobz-modules/odoo/odoo/http.py", line 766, in _handle_exception
    return super(HttpRequest, self)._handle_exception(exception)
  File "/opt/openerp/code/openerp/project/trobz-modules/odoo/odoo/addons/base/ir/ir_http.py", line 195, in _dispatch
    result = request.dispatch()
  File "/opt/openerp/code/openerp/project/trobz-modules/odoo/odoo/http.py", line 825, in dispatch
    r = self._call_function(**self.params)
  File "/opt/openerp/code/openerp/project/trobz-modules/odoo/odoo/http.py", line 331, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/opt/openerp/code/openerp/project/trobz-modules/odoo/odoo/service/model.py", line 119, in wrapper
    return f(dbname, *args, **kwargs)
  File "/opt/openerp/code/openerp/project/trobz-modules/odoo/odoo/http.py", line 324, in checked_call
    result = self.endpoint(*a, **kw)
  File "/opt/openerp/code/openerp/project/trobz-modules/odoo/odoo/http.py", line 933, in __call__
    return self.method(*args, **kw)
  File "/opt/openerp/code/openerp/project/trobz-modules/odoo/odoo/http.py", line 504, in response_wrap
    response = f(*args, **kw)
  File "/opt/openerp/code/openerp/project/trobz-modules/addons/vertical-medical/website_medical_prescription_order_line/controllers/main.py", line 23, in my_medical
    response = super(WebsiteMedical, self).my_medical()
  File "/opt/openerp/code/openerp/project/trobz-modules/odoo/odoo/http.py", line 504, in response_wrap
    response = f(*args, **kw)
  File "/opt/openerp/code/openerp/project/trobz-modules/addons/vertical-medical/website_medical_patient/controllers/main.py", line 23, in my_medical
    response = super(WebsiteMedical, self).my_medical()
  File "/opt/openerp/code/openerp/project/trobz-modules/odoo/odoo/http.py", line 504, in response_wrap
    response = f(*args, **kw)
  File "/opt/openerp/code/openerp/project/trobz-modules/addons/vertical-medical/website_medical/controllers/main.py", line 170, in my_medical
    return request.website.render("website_medical.medical_main", values)
AttributeError: 'website' object has no attribute 'render'
```

**Comment by lasley** [comment]
Thanks for reporting this @jcdrubay - please confirm this is fixed in #83 

I had to inherit the medical entity from `mail.thread` so that the wires wouldn't get crossed with the partner thread. Added benefit is now all the medical entities have threads!

**Comment by jcdrubay** [comment]
Issue is fixed :)
